crates/relayer/src/main.rs

```diff
@@ -175,6 +178,15 @@ type AppState = mpsc::Sender<RequestPayload>;
 type RequestPayload = (RelayRequest<'static>, HandlerSender);
 type HandlerSender = oneshot::Sender<Result<EyreResult<Vec<u8>>, ServerError>>;
 
+/// Health check endpoint
+async fn health_check() -> impl IntoResponse {
+    Json(serde_json::json!({
+        "status": "healthy",
+        "service": "calimero-relayer",
+        "timestamp": chrono::Utc::now().to_rfc3339()
```

Comment:
Please add this part to the `DOCKER_RELAYER.md` - specify that the health check returns the timestamp in RFC-3339 format.